index.html

```diff
@@ -421,19 +383,91 @@
 
 
 
+            </div><!--MAIN ROW CLOSE-->
 
 
+            <!--***************** MAIN CONTENT *******************-->
 
+            <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
+            <script>
+                var username = 'eddyj9';
 
+                var baseUrl = 'https://api.github.com/users/' + username;
+                var repoUrl = baseUrl + '/repos';
+                var repoArea = $('#repo_area');
 
-        </div><!--MAIN ROW CLOSE-->
+                $.ajax({
+                    method: 'GET',
+                    url: baseUrl,
+                    dataType: 'json'
+                }).done(function (userInformation) {
+                    console.log("DONE");
 
+                    $('#user_email').replaceWith('eduardojimenez9@yahoo.com');
+                    $('#user_name').replaceWith('<h3>' + username + '</h3>');
+                    $('#user_location').replaceWith(userInformation.location);
+                    $('#profile_name').replaceWith('<h2>' + userInformation.name + '</h2>');
+                    $('#user_html_url').replaceWith(userInformation.html_url);
+                    $('#profile_image').replaceWith('<img src='+userInformation.avatar_url+' height="215" width="215">');
+                });
 
-        <!--***************** MAIN CONTENT *******************-->
+                $.ajax({
+                    method: 'GET',
+                    url: repoUrl,
+                    dataType: 'json'
+                }).done(function (repoInformation) {
+                    console.log("Repo Done");
+                    var repos = repoInformation;
+                    if(repos === 0){
+                        repoArea.html('<h1> NO REPO FOUND! </h1>');
+                    } else{
+                        repos.forEach(function(repo) {
+                            console.log("looping for repos");
+                            var repoTitle = repo.name;
+                            var repoDesc = repo.description;
+                            var repoCode = '<div class="row tab"><!-- THIRD ROW -->\
```

Comment:
We found that you can use \` to surround the block of html which prevents the need for \ on each line.  Additionally, `${VAR}` will swap in the variable without having to break up the line.